recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
This should be ready to go. Usually self-merges in feedstocks are fair game. Let me know if you want me to merge this for you though.